treeherder/autoclassify/detect_intermittents.py

```diff
@@ -1,8 +1,8 @@
 import logging
```

Comment:
You could have opened another pull request for the removal, I wasn't saying that it had to happen as part of this PR.